rpcs3/Emu/Cell/SPURecompiler.cpp

```diff
@@ -4474,6 +4474,11 @@ class spu_llvm_recompiler : public spu_recompiler_base, public cpu_translator
 
 		// Store instructions
 		std::array<llvm::StoreInst*, s_reg_max> store{};
+
+		// Store reordering/elimination protection
+		std::array<usz, s_reg_max> store_context_last_id = fill_array<usz>(0); // Protects against illegal forward ordering
```

Comment:
```suggestion
		std::array<usz, s_reg_max> store_context_last_id{}; // Protects against illegal forward ordering
```

Reply:
Dunno what is wrong with explicitly, this is all constexpr anyways so compiler output should be the same.